go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/posthog/posthog-go v1.2.19
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.9.0
```

Comment:
This seems to be pretty popular in the Go ecosystem, but I defer to you @jpmcb for what you recommend for tests.